package.json

```diff
@@ -66,9 +66,8 @@
 			"activitybar": [
 				{
 					"id": "hexExplorer",
-					"title": "Hex Editor Panel",
-					"icon": "panel-icon.svg",
-					"when": "hexEditor:openEditor"

```

Comment:
Is there any repercussions from removing the when condition that hides and shows this view when a hex editor is open

Reply:
Good catch, that was accidentally removed in bad merge conflict fix